### ISSUE TITLE
Class JSFile not found

### ISSUE THREAD

**Issue by dominique-unruh** [reasonml-editor/reasonml-idea-plugin#276]
plugin version: 0.93.1-2020.2

### Description

The following exception is repeatedly reported in the "IDE Fatal Errors" dialog of IDEA:
```
java.lang.NoClassDefFoundError: com/intellij/lang/javascript/psi/JSFile
	at com.reason.ide.IconProvider.isBsJsFile(IconProvider.java:92)
	at com.reason.ide.IconProvider.getIcon(IconProvider.java:42)
	at com.intellij.ide.projectView.impl.CompoundIconProvider.getIcon(CompoundIconProvider.java:32)
	at com.intellij.ide.projectView.impl.CompoundIconProvider.findIcon(CompoundIconProvider.java:55)
	at com.intellij.analysis.problemsView.toolWindow.FileNode.update(FileNode.kt:23)
	at com.intellij.analysis.problemsView.toolWindow.Node.update(Node.kt:27)
	at com.intellij.ide.util.treeView.PresentableNodeDescriptor.getUpdatedPresentation(PresentableNodeDescriptor.java:74)
	at com.intellij.ide.util.treeView.PresentableNodeDescriptor.update(PresentableNodeDescriptor.java:27)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
	at com.intellij.analysis.problemsView.toolWindow.ProblemsTreeModel.getChildren(ProblemsTreeModel.java:55)
	at com.intellij.ui.tree.AsyncTreeModel$CmdGetChildren.getNode(AsyncTreeModel.java:583)
	at com.intellij.ui.tree.AsyncTreeModel$Command.get(AsyncTreeModel.java:478)
	at com.intellij.ui.tree.AsyncTreeModel$Command.get(AsyncTreeModel.java:444)
	at com.intellij.util.concurrency.Invoker$Task.run(Invoker.java:316)
	at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:1110)
	at com.intellij.openapi.progress.util.ProgressIndicatorUtils.lambda$runInReadActionWithWriteActionPriority$0(ProgressIndicatorUtils.java:77)
	at com.intellij.openapi.progress.util.ProgressIndicatorUtils.runActionAndCancelBeforeWrite(ProgressIndicatorUtils.java:153)
	at com.intellij.openapi.progress.util.ProgressIndicatorUtils.lambda$runWithWriteActionPriority$1(ProgressIndicatorUtils.java:114)
	at com.intellij.openapi.progress.ProgressManager.lambda$runProcess$0(ProgressManager.java:56)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$2(CoreProgressManager.java:170)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:629)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:581)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:60)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:157)
	at com.intellij.openapi.progress.ProgressManager.runProcess(ProgressManager.java:56)
	at com.intellij.openapi.progress.util.ProgressIndicatorUtils.runWithWriteActionPriority(ProgressIndicatorUtils.java:111)
	at com.intellij.openapi.progress.util.ProgressIndicatorUtils.runInReadActionWithWriteActionPriority(ProgressIndicatorUtils.java:77)
	at com.intellij.util.concurrency.Invoker.invokeSafely(Invoker.java:205)
	at com.intellij.util.concurrency.Invoker.lambda$offerSafely$0(Invoker.java:183)
	at com.intellij.util.concurrency.Invoker$Background.lambda$offer$0(Invoker.java:541)
	at com.intellij.util.concurrency.BoundedTaskExecutor.doRun(BoundedTaskExecutor.java:215)
	at com.intellij.util.concurrency.BoundedTaskExecutor.access$200(BoundedTaskExecutor.java:26)
	at com.intellij.util.concurrency.BoundedTaskExecutor$1.execute(BoundedTaskExecutor.java:194)
	at com.intellij.util.ConcurrencyUtil.runUnderThreadName(ConcurrencyUtil.java:207)
	at com.intellij.util.concurrency.BoundedTaskExecutor$1.run(BoundedTaskExecutor.java:183)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:668)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:665)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1.run(Executors.java:665)
	at java.base/java.lang.Thread.run(Thread.java:834)
```

**Comment by giraud** [comment]
what product are you using ?

**Comment by dominique-unruh** [comment]
I assume which Intelli/J IDEA variant? Below is what the about dialog says:

IntelliJ IDEA 2020.2.3 (Community Edition)
Build #IC-202.7660.26, built on October 6, 2020
Runtime version: 11.0.8+10-b944.34 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
Linux 5.4.0-48-generic
GC: ParNew, ConcurrentMarkSweep
Memory: 928M
Cores: 8
Non-Bundled Plugins: Batch Scripts Support, CMD Support, org.jetbrains.idea.grammar, reasonml, org.jetbrains.plugins.hocon, name.kropp.intellij.makefile, org.jetbrains.kotlin, org.intellij.scala, PythonCore
Current Desktop: ubuntu:GNOME

**Comment by giraud** [comment]
Fixed in 0.94
